client/src/curriculum/index.tsx

```diff
@@ -1,26 +1,26 @@
-import { Route, Routes } from "react-router-dom";
-
 import { HydrationData } from "../../../libs/types/hydration";
 import { CurriculumModuleOverview } from "./overview";
 import { CurriculumModule } from "./module";
 import { CurriculumAbout } from "./about";
 import { CurriculumLanding } from "./landing";
 
 import "./index.scss";
+import { CurriculumData, CurriculumDoc } from "../../../libs/types/curriculum";
+import { Template, useCurriculumDoc } from "./utils";
+import { CurriculumDefault } from "./default";
 
-export function Curriculum(appProps: HydrationData) {
-  return (
-    <Routes>
-      <Route path="/" element={<CurriculumLanding {...appProps} />} />
-      <Route
-        path="/about-curriculum/"
-        element={<CurriculumAbout {...appProps} />}
-      />
-      <Route
-        path="/:module/"
-        element={<CurriculumModuleOverview {...appProps} />}
-      />
-      <Route path="/:module/*" element={<CurriculumModule {...appProps} />} />
-    </Routes>
-  );
+export function Curriculum(appProps: HydrationData<any, CurriculumDoc>) {
+  const doc = useCurriculumDoc(appProps as CurriculumData);
+  switch (doc?.template) {
+    case Template.Landing:
+      return <CurriculumLanding {...appProps} />;
+    case Template.Overview:
+      return <CurriculumModuleOverview {...appProps} />;
+    case Template.Module:
+      return <CurriculumModule {...appProps} />;
+    case Template.About:
+      return <CurriculumAbout {...appProps} />;
+    default:
+      return <CurriculumDefault {...appProps} />;
```

Comment:
Can you add a comment about what pages are shown with this default layout?

```suggestion
      // ...
      return <CurriculumDefault {...appProps} />;
```

Reply:
Added
```
case Template.Default:
```